pipeline/container/manager.py

```diff
@@ -31,7 +31,7 @@ def _get_url_or_path(input_schema: run_schemas.RunInput) -> str | None:
 
 
 class Manager:
-    def __init__(self, pipeline_path: str):
+    def _load(self, pipeline_path: str):
```

Comment:
maybe we could add `with logger.contextualize(pipeline_stage="loading"):` below here? see comment below

Reply:
sounds great